Project/src/MakeCall/CallCard.js

```diff
@@ -47,7 +48,8 @@ export default class CallCard extends React.Component {
             dominantSpeakerMode: false,
             dominantRemoteParticipant: undefined,
             remoteVolumeIndicator: undefined,
-            remoteVolumeLevel: undefined
+            remoteVolumeLevel: undefined,
+            logMediaStats: false
```

Comment:
would we rather want to start/stop collector based on the flag? to avoid running stats in background?

Reply:
we didn't start/stop the collector. it's only for logging control because raw mediastats sample event emits once per second and it goes to log file buffer.